client/my-sites/themes/install-theme-button.tsx

```diff
@@ -68,7 +77,7 @@ const InstallThemeButton = ( {
 	);
 };
 
-const mapStateToProps = ( state ) => {
+const mapStateToProps = ( state: IAppState ) => {
```

Comment:
I'm not familiar with the current policies in Calypso, so take this with a grain of salt, but we tend to avoid Hungarian-prefix styling of type names. There's no need for the leading `I` - the information about the type should be readily available in any supporting IDE, and it's also obvious via syntax here as a type following the `state:` param. Note too that it doesn't _need_ to be an interface, so if we change it to a `type` later (possibly via type union or intersection) then our name will be out of sync with the reality in a way that never needed to happen.